suitcase/fields.py

```diff
@@ -8,6 +8,7 @@
 import six
 from suitcase.exceptions import SuitcaseChecksumException, SuitcaseProgrammingError, \
     SuitcaseParseError, SuitcaseException, SuitcasePackStructException
+import suitcase
```

Comment:
I agree that it would look nicer, but it introduces a circular dependency between fields.py and structure.py.